tests/test_main.py

```diff
@@ -465,16 +428,12 @@
 @pytest.mark.django_db
 def test_movie_session_service_get_movie_session_by_date(database_data):
     sessions_1 = get_movies_sessions("2019-8-19")
-    assert list(sessions_1.values_list(
-        "movie__title", "cinema_hall__name"
-    )) == [
+    assert list(sessions_1.values_list("movie__title", "cinema_hall__name")) == [
         ("Matrix", "Blue")
     ]
 
     sessions_2 = get_movies_sessions("2021-4-3")
```

Comment:
The `get_movies_sessions` function expects a `datetime` object for the `session_date` parameter, but a string is provided here. Consider converting the string to a `datetime` object before passing it to the function.